docs/proposals/keeping-transient-pod-name-spaces-alive.md

```diff
@@ -0,0 +1,54 @@
+# Keeping transient pod name spaces alive
+
+## Introduction
+The goal of this proposal is to give administrators options to keep namespaces and therefore APB pods after the execution. The use cases for this feature are demos and debugging APBs.
+
+## Problem Description
+The [bug](https://bugzilla.redhat.com/show_bug.cgi?id=1497766) was created when we moved to creating transient namespaces during execution of the APB pod. What happens  This bug creates issues for debugging APB's as well as issues with demos. 
```

Comment:
TYPO: "What happens  This bug"  -> This bug creates ... 
Also what sort of problems are created? How about:

[Bug 1497766](https://bugzilla.redhat.com/show_bug.cgi?id=1497766) titled "APB Pods are deleted even when an error occurs" occurred when we moved to creating transient namespaces during the APB pod execution. This bug makes it difficult to debug APB development and creating demos because the pods do not live long enough to capture useful information.